platform/logging/appenders/console/ConsoleAppender.ts

```diff
@@ -0,0 +1,14 @@
+import { LogRecord } from '../../LogRecord';
+import { BaseAppender } from '../base/BaseAppender';
+import { ConsoleAppenderConfig } from './ConsoleAppenderConfig';
+
+export class ConsoleAppender extends BaseAppender {
+  constructor(protected readonly config: ConsoleAppenderConfig) {
+    super(config)
+  }
+
+  append(record: LogRecord) {
+    super.append(record);
+    console.log(this.logRecordToFormattedString(record));
```

Comment:
Do we have a plan what and how we want to colorize? It seems ES doesn't do this, but let me think...